endpointmanager/pkg/capabilityhandler/capabilityreceiver.go

```diff
@@ -21,6 +21,10 @@ func formatMessage(message []byte) (*endpointmanager.FHIREndpoint, error) {
 		return nil, err
 	}
 
+	errs, ok := msgJSON["err"].(string)
+	if ok && len(errs) > 0 {
+		log.Warn(errs)
+	}
```

Comment:
i think it makes sense to just capture these errors in the db. i don't think it makes sense to log them here because this is not where the request was made. also, recommend checking `ok` to make sure it could be serialized to a string.

recommend just this:
```
	errs, ok := msgJSON["err"].(string)
	if !ok {
		return nil, fmt.Errorf("unable to cast message Error to string")
	}
```